content/notes/config.md

```diff
@@ -23,6 +24,49 @@ links: # Links to show in footer
     link: https://github.com/jackyzha0
 ```
 
+### Code Block Titles
+
+Quartz relies on Hugo's code highlighter,
```

Comment:
I would dare say that this pre-amble is probably unnecessary! Just the last sentence 'To add code titles with Quartz:' is probably sufficient!

Reply:
You're right! I resolved this locally.